antarest/study/storage/rawstudy/model/filesystem/json_file_node.py

```diff
@@ -17,7 +17,9 @@
 
 
 class JsonReader(IReader):
-    def read(self, path: Path) -> JSON:
+    def read(self, path: Any) -> JSON:
+        if not isinstance(path, Path):
+            return cast(JSON, json.load(path))
         with open(path, "r") as fh:
             return cast(JSON, json.load(fh))
 
```

Comment:
- Inverser la condition (pour se passer du `not`)
- utiliser `json.loads` et lire le fichier avec `path.read_text(encoding="utf-8")`